pkg/ask/ask.go

```diff
@@ -0,0 +1,36 @@
+package ask
+
+import (
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/algolia/cli/pkg/utils"
+)
+
+func AskInputQuestion(message string, storage *string, defaultValue string, opts ...survey.AskOpt) error {
+	return survey.AskOne(&survey.Input{
+		Message: message,
+		Default: defaultValue,
+	}, storage, opts...)
+}
+
+func AskCommaSeparatedInputQuestion(message string, storage *[]string, defaultValues []string, opts ...survey.AskOpt) error {
```

Comment:
Survey package provide an interface for custom types that we could follow: https://github.com/AlecAivazis/survey#custom-types

Reply:
Nice! 🙏